inspirehep/modules/theme/bundles.py

```diff
@@ -63,9 +63,10 @@
         "toastr": "~2.1.2",
         "clipboard": "~1.5.8",
         "flightjs": "~1.5.0",
-        "angular": "~1.4.8",
-        "inspirehep-js": "git+https://git@github.com/inspirehep/inspirehep-js.git"
-
+        "angular": "~1.4.9",
+        "angular-sanitize": "~1.4.9",
+        "inspirehep-js": "git+https://git@github.com/inspirehep/inspirehep-js.git",
+        "ngclipboard": "~1.1.1"
```

Comment:
This file does not need to be modified. `ngclipboard ` is a dependency of `inspirehep-js` so it should be defined in the [package.json](https://github.com/inspirehep/inspirehep-js/blob/master/package.json). Then it will get installed correctly.